Tests/ComposableArchitectureTests/EffectFailureTests.swift

```diff
@@ -1,58 +1,66 @@
 #if DEBUG
-  import Combine
-  import ComposableArchitecture
-  import XCTest
-
-  @MainActor
-  final class EffectFailureTests: XCTestCase {
-    var cancellables: Set<AnyCancellable> = []
+#if canImport(OpenCombine)
+import OpenCombine
+#else
+import Combine
+#endif
+import ComposableArchitecture
+import XCTest
 
-    func testTaskUnexpectedThrows() async {
-      guard #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) else { return }
+@MainActor
+final class EffectFailureTests: XCTestCase {
+  var cancellables: Set<AnyCancellable> = []
 
-      var line: UInt!
-      XCTExpectFailure {
-        $0.compactDescription == """
-          An "EffectTask.task" returned from "\(#fileID):\(line+1)" threw an unhandled error. …
+  func testTaskUnexpectedThrows() async throws {
+    guard #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) else { return }
+    try XCTSkipIfWindowsExpectFailure()
 
-              EffectFailureTests.Unexpected()
+    var line: UInt!
+    XCTExpectFailure {
+      $0.compactDescription
+        == """
+        An "EffectTask.task" returned from "\(#fileID):\(line+1)" threw an unhandled error. …
 
-          All non-cancellation errors must be explicitly handled via the "catch" parameter on \
-          "EffectTask.task", or via a "do" block.
-          """
-      }
+            EffectFailureTests.Unexpected()
 
-      line = #line
-      let effect = EffectTask<Void>.task {
-        struct Unexpected: Error {}
-        throw Unexpected()
-      }
+        All non-cancellation errors must be explicitly handled via the "catch" parameter on \
+        "EffectTask.task", or via a "do" block.
+        """
+    }
 
-      for await _ in effect.values {}
+    line = #line
+    let effect = EffectTask<Void>.task {
+      struct Unexpected: Error {}
+      throw Unexpected()
     }
 
-    func testRunUnexpectedThrows() async {
-      guard #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) else { return }
+    for await _ in effect.values {}
+  }
 
-      var line: UInt!
-      XCTExpectFailure {
-        $0.compactDescription == """
-          An "EffectTask.run" returned from "\(#fileID):\(line+1)" threw an unhandled error. …
+  func testRunUnexpectedThrows() async throws {
+    guard #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) else { return }
+    try XCTSkipIfWindowsExpectFailure()
 
-              EffectFailureTests.Unexpected()
+    var line: UInt!
+    XCTExpectFailure {
+      $0.compactDescription
+        == """
```

Comment:
[![ditto](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/jlrqNw1yzCqqR4b6AxWv/23d915f9-5eff-435f-a181-a745c450de1b/ditto.png)](https://app.graphite.dev/macros/thebrowsercompany)

Reply:
same indentation issue as mentioned above, will give it a shot to see if I can fix it to make the diff smaller